homeassistant/config.py

```diff
@@ -50,6 +50,9 @@
     ('ios.conf', '.ios.conf'),
 )
 
+CORE_STORAGE_KEY = 'homeassistant.storage'
```

Comment:
Not sure if this is a good name, or it should be `'core.storage'`, `'homeassistant.core'` or something else?

Reply:
Let's do core config. 

```suggestion
CORE_STORAGE_KEY = 'homeassistant.core_config'
```